junit/MatrixTest.java

```diff
@@ -69,8 +69,21 @@ public void tooShort(){
         assertEquals(array2, m.getArray());
     }
     //Matrix(double[] vals, int m){}
-
-    //Matrix(int m, int n){}
+    
+    /**
+     * testMatrixConstructor3 -Tests the third matrix constructor as specified in issue #7.
+    **/
+    @Test
+    public void testMatrixConstructor3() {
```

Comment:
Since it isn't specified in the API docs, how should we handle being passed negative numbers or zero? We should decide on some behavior and then write tests to ensure that we handle it properly.